x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/connector_add_flyout.tsx

```diff
@@ -281,11 +288,32 @@ const upgradeYourLicenseCallOut = (
       defaultMessage="With an upgraded license, you have the option to connect to more 3rd party services."
     />
     <EuiSpacer size="xs" />
```

Comment:
Can we change this size to `s` or even `m` so the message isn't so close to the buttons?

Reply:
Sure 👍

Reply:
changed to `s` as `m` felt too wide